Allow custom name overrides

syrifgit opened the pull request:
Simple implementation to allow overriding names!

<img width="1725" height="1086" alt="image" src="https://github.com/user-attachments/assets/5743e083-e256-4767-9a54-69764608347d" />

<img width="1312" height="1057" alt="image" src="https://github.com/user-attachments/assets/64de271d-3c32-407a-b896-763d78d9cb5d" />

jcbmcn commented:
@syrifgit, excellent addition!
2 things:

- (minor, I'm just nit picky) Can you move the order for the `Enable Custom Name` checkbox above the `Custom Names` text box. I feel that would flow a bit better, and reduce confusion on why their custom names aren't working (some people don't scroll down far enough to read.)
- Add a short description to the README with this change. 

Once those have been implemented, I'm happy to get this merged in and I'll go work with the plugin admins to get this updated.

syrifgit commented:
Just added new commit.
- Swapped order
- Changed the Kourend example to `Civitas illa Fortis` to better demonstrate to the user that spaces in names are fine
- Added instructions to readme

<img width="347" height="358" alt="image" src="https://github.com/user-attachments/assets/afc4c30a-14fc-4a7f-b6f3-b6c3ec291681" />

jcbmcn commented:
Excellent!

syrifgit commented:
No problem! To be fair it's because I wanted the feature for myself lol
